### ISSUE TITLE
Nest `CombineReferences` inside `WriteCombinedReferences`

### ISSUE THREAD

**Issue by cisaacstern** [pangeo-forge/pangeo-forge-recipes#634]
Working on #610 has me thinking about how to easily communicate common recipe styles. Over there, I'm centering on the idea that the basic version of all common pipeline styles should be expressible as: `FilePattern | Opener | Writer`. `StoreToZarr` fits this style, but our kerchunk recipes do not:

https://github.com/pangeo-forge/pangeo-forge-recipes/blob/f6af0e03725caf93691cda4fc5724159a2671898/examples/feedstock/hrrr_kerchunk_concat_step.py#L48-L61

To align kerchunk recipes with this style, we can nest `CombineReferences` inside `WriteCombinedReferences`, making the basic kerchunk recipe expressible as:

```python

recipe = (
    beam.Create(pattern.items())
    | OpenWithKerchunk(...)
    | WriteCombinedReferences(...)
)
```

**Comment by norlandrhagen** [comment]
This seems like a good call!

**Comment by norlandrhagen** [comment]
Closed by #635
